src/components/Recommend/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import * as _ from './style';
+
+export const Recommend = () => {
+  return (
+    <_.RecommendContainer>
+      <_.RecommendWrap>
+        <_.RecommendTitleWrap>
```

Comment:
```suggestion
        <_.RecommendTitle>
```

`Wrap`을 postfix로 붙이시는 이유가 있으신가요? 기준이 있는지 궁금합니당~

Reply:
저도 궁금합니다!